generator/source/release/bumper/notes.md

```diff
@@ -0,0 +1,322 @@
+  API. This CLI will let us test the JSON-RPC API, and will eventually allow us
+  to deprecate the existing REST-API-based CLI, which is now available in the
+  `seth-cli-go` container.
+- The existing JSON-RPC API has been updated to align more closely with
+  existing Ethereum JSON-RPC implementations, particularly in how it handles
+  account management. This change provides better inter-compatibility with
+  off-the-shelf Ethereum tooling.
+- The Burrow version was updated from 0.17 to 0.21 and vendor dependencies were
+  removed.
+- The build process and dependencies have been updated and aligned with current
+  best practices, such as formatting the Rust code and linting it with Clippy.
+- The documentation has been updated with minor corrections.
+- Several minor bugs have been fixed, such as a segfault occurring when
+  creating an account with a nonce, and invalid addresses being displayed in
+  contract listing.
+- This release includes Dockerfiles suitable for publishing to Docker Hub.
+
+## Sawtooth Sabre
+
+Hyperledger Sawtooth 1.1 supports [Sawtooth
+Sabre](https://github.com/hyperledger/sawtooth-sabre), a transaction family
+that implements on-chain smart contracts executed in a WebAssembly virtual
+machine. Sabre smart contracts are stored on chain and executed using the Sabre
+transaction processor.
+
+Sawtooth Sabre includes an SDK for writing Sabre smart contracts in Rust. The
+smart contracts can be written in such a way that they can be compiled into
+transaction processor and run without Sabre. This also makes it easy to convert
+already-written Rust transaction processors to a Sabre smart contract.
+
+**Note:** Sabre is currently at version 0.1 and is under active development.
+
+[> back to top <](#top)
+
+# DevOps and CI
+
+## Build System Improvements
+
+- The `bin/build_all` script and related build scripts have been replaced with
+  `docker-compose`.
+- The Docker Compose files now have 'build' sections, so that pre-building
+  steps aren't necessary and all required images can be built with a single
+  `docker-compose up` command.
+- The `sawtooth-dev-{lang}` Dockerfiles are deprecated. Each component now has
+  its own Dockerfile for development.
+
+Check out the [build system
+RFC](https://github.com/hyperledger/sawtooth-rfcs/pull/25) for more details.
+
+**Note**: Requires Docker Engine 18.02.0 or later.
+
+## Docker
+
+- Each component now has "installed" Dockerfiles that utilize multi-stage
+  builds. These Dockerfiles are suitable for publishing to a docker registry.
+
+**Note**: Requires Docker Engine 18.02.0 or later.
+
+## Kubernetes
+
+- This release includes example files for two Kubernetes deployments: A
+  five-node network using PoET simulator and a single-node environment using
+  dev mode consensus. The Application Developer’s Guide describes how to use
+  these example files; see the [Hyperledger Sawtooth
+  documentation](#hyperledger-sawtooth-documentation).
```

Comment:
Bad link.  It should be this link instead:

https://sawtooth.hyperledger.org/docs/core/nightly/master/app_developers_guide.html

Reply:
Except use the right "latest" version of that link.  Sorry!

Reply:
Link href set to https://sawtooth.hyperledger.org/docs/core/releases/latest/app_developers_guide.html